pkg/argocd/argocd_test.go

```diff
@@ -702,3 +722,10 @@ func TestKubernetesClient_UpdateSpec_Conflict(t *testing.T) {
 
 	assert.Equal(t, "https://github.com/argoproj/argocd-example-apps", spec.Source.RepoURL)
 }
+
+func Test_parseImageList(t *testing.T) {
+	assert.Equal(t, []string{"foo", "bar"}, parseImageList(" foo, bar ").Originals())
+	// should whitespace inside the spec be preserved?
```

Comment:
I think whitespace stripping should be done here, but let's do that in another PR: https://github.com/argoproj-labs/argocd-image-updater/blob/f1cc8e4542b0531f66fc5546f7467833b1d08ebe/pkg/image/image.go#L133